dictionaries/CallMap_historical.php

```diff
@@ -3340,28 +3340,27 @@
     'IntlCodePointBreakIterator::preceding' => ['int', 'offset'=>'string'],
     'IntlCodePointBreakIterator::previous' => ['int'],
     'IntlCodePointBreakIterator::setText' => ['bool', 'text'=>'string'],
-    'IntlDateFormatter::__construct' => ['void', 'locale'=>'?string', 'datetype'=>'?int', 'timetype'=>'?int', 'timezone='=>'null|string|IntlTimeZone|DateTimeZone', 'calendar='=>'null|int|IntlCalendar', 'pattern='=>'string'],
-    'IntlDateFormatter::create' => ['IntlDateFormatter|false', 'locale'=>'?string', 'datetype'=>'?int', 'timetype'=>'?int', 'timezone='=>'null|string|IntlTimeZone|DateTimeZone', 'calendar='=>'int|IntlCalendar', 'pattern='=>'string'],
-    'IntlDateFormatter::format' => ['string|false', 'args'=>''],
-    'IntlDateFormatter::formatObject' => ['string|false', 'object'=>'object', 'format='=>'mixed', 'locale='=>'string'],
+    'IntlDateFormatter::__construct' => ['void', 'locale'=>'?string', 'datetype'=>'null|int', 'timetype'=>'null|int', 'timezone='=>'IntlTimeZone|DateTimeZone|string|null', 'calendar='=>'IntlCalendar|int|null', 'pattern='=>'?string'],
+    'IntlDateFormatter::create' => ['?IntlDateFormatter', 'locale'=>'?string', 'datetype'=>'null|int', 'timetype'=>'null|int', 'timezone='=>'IntlTimeZone|DateTimeZone|string|null', 'calendar='=>'IntlCalendar|int|null', 'pattern='=>'?string'],
+    'IntlDateFormatter::format' => ['string|false', 'value'=>'IntlCalendar|DateTime|array{0: int, 1: int, 2: int, 3: int, 4: int, 5: int, 6: int, 7: int, 8: int}|array{tm_sec: int, tm_min: int, tm_hour: int, tm_mday: int, tm_mon: int, tm_year: int, tm_wday: int, tm_yday: int, tm_isdst: int}|string|int|float'],
+    'IntlDateFormatter::formatObject' => ['string|false', 'object'=>'IntlCalendar|DateTime', 'format='=>'array{0: int, 1: int}|int|string|null', 'locale='=>'?string'],
     'IntlDateFormatter::getCalendar' => ['int'],
     'IntlDateFormatter::getCalendarObject' => ['IntlCalendar'],
     'IntlDateFormatter::getDateType' => ['int'],
     'IntlDateFormatter::getErrorCode' => ['int'],
     'IntlDateFormatter::getErrorMessage' => ['string'],
-    'IntlDateFormatter::getLocale' => ['string|false'],
+    'IntlDateFormatter::getLocale' => ['string', 'which='=>'int'],
```

Comment:
The doc doesn't have a `which` param: https://www.php.net/manual/fr/intldateformatter.getlocale.php

Is this a doc error?

Reply:
nvm, you changed the name on more recent versions :)

Reply:
Yes, the parameter changed approximatively when PHP 8.0 rolled out.

See a former documentation of the exact same page from May 2020 for instance: https://web.archive.org/web/20200502000358/https://www.php.net/manual/fr/intldateformatter.getlocale.php